source/features/faster-pr-diff-options.tsx

```diff
@@ -90,31 +82,44 @@ function init(): false | void {
 		prTitle.title = prTitle.textContent!;
 	}
 
-	// Remove previous options UI
-	const singleCommitUI = select('[data-ga-load^="Diff, view"]');
+	// Only show the native dropdown on medium and small screens #2597
+	select('.js-diff-settings')!.closest('details')!.classList.add('d-lg-none');
 
-	if (singleCommitUI) {
-		singleCommitUI.remove();
-		return;
-	}
+	// Make space for the new button by removing "Changes from" #655
+	(select('.js-commits-filtered') ?? select('[data-hotkey="c"]'))!.firstChild!.remove();
 
-	const prUI = select('.js-diff-settings');
-	if (prUI) {
-		prUI.closest('details')!.remove();
+	// Remove extraneous padding around "Clear filters" button
+	select('.subset-files-tab')?.classList.replace('px-sm-3', 'ml-2');
```

Comment:
We need to preserve the media query, I'm not sure if this is right:

```suggestion
	select('.subset-files-tab')?.classList.replace('px-sm-3', 'ml-sm-2');
```

Without this, we're _adding_ margin on windows <544px